test/ui-test/testSuites/suite_onboarding/tst_signUpNegativeCases/test.feature

```diff
@@ -10,77 +10,43 @@ Feature: Status Desktop Sign Up, negative cases
    [Cleanup]
    ** the user navigates to first onboarding page
 
-  Scenario Outline: The user cannot sign up with wrong username format
-    Given the user clears input "onboarding_DiplayName_Input"
-    When the user inputs the following "<username>" with ui-component "onboarding_DiplayName_Input"
-    Then the following ui-component "onboarding_DetailsView_NextButton" is not enabled
+  #Consider passing examples in a datatable to cut overhead
+  Scenario Outline: Entering a <reason> username
+    Given the user clears the display name field
```

Comment:
My preference is to avoid lower level component names in steps. We could do `Given the user clears the "<field name>" field` though.

In this case "display name" would be sent and then mapped to the component on the back end via the step file and maybe some helper method. This uncouples it from the code and avoids having to update every gherkin step working with that component if it changes somehow.